includes/genotypes_loader.vcf.inc

```diff
@@ -51,6 +51,11 @@ function genotypes_loader_load_VCF($input_file = NULL, $options, $types) {
   $select_only = 0;
   $insert_only = 1;
   $both = 2;
+  
+  // Set the file name to be used for remote copy
+  if ($options['storage_method'] == 'genotype_call') {
+    $remote_copy_filename = '/tmp/genotypes_loader.remotecopy.csv';
```

Comment:
For consistency, I would follow the same convention that you use at https://github.com/UofS-Pulse-Binfo/genotypes_loader/pull/19/files?utf8=%E2%9C%93&diff=unified&w=1#diff-eb8a4ddc56c5aa596c7c31e403e0f285R138

Reply:
Done in commit https://github.com/UofS-Pulse-Binfo/genotypes_loader/commit/034a87b795cc7478c99f82ae501c15d5b18bf5b5. I opted to keep only the table name since I knew in this particular case it would be redundant (table name and record name would both be 'genotype_call'). I am still opting to use both table and record in the API function since I want to keep that function generic for the future and thus they are both important to ensure uniqueness.